homeassistant/components/alexa/__init__.py

```diff
@@ -87,18 +87,12 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
 
     config = config[DOMAIN]
 
-    flash_briefings_config = config.get(CONF_FLASH_BRIEFINGS)
-
     intent.async_setup(hass)
 
-    if flash_briefings_config:
+    if (flash_briefings_config := config.get(CONF_FLASH_BRIEFINGS)) is not None:
         flash_briefings.async_setup(hass, flash_briefings_config)
 
-    try:
-        smart_home_config = config[CONF_SMART_HOME]
-    except KeyError:
-        pass
-    else:
+    if (smart_home_config := config.get(CONF_SMART_HOME)) is not None:
```

Comment:
You dont need the is not none in either cases i think. None is a false value. As long as you dont need to explicitly pass in an empty dict/list into these functions, you should be fine with `if x:= bla.get():`

Reply:
True, thanks!